libs/membership/api/src/lib/dashboard/invoices/dashboard-invoice.model.ts

```diff
@@ -0,0 +1,16 @@
+import {Field, Int, ObjectType} from '@nestjs/graphql'
+
```

Comment:
Even longer path. How about libs/membership/models/dashboard-invoice.model.ts?

Reply:
That reduces cohesivenees. To change the query you would then need to change multiple directories.